packages/roosterjs-content-model-dom/lib/domToModel/utils/isBlockElement.ts

```diff
@@ -1,6 +1,6 @@
 import { getDefaultStyle } from './getDefaultStyle';
 
-const BLOCK_DISPLAY_STYLES = ['block', 'list-item', 'table', 'table-cell', 'flex'];
+const BLOCK_DISPLAY_STYLES = ['block', 'list-item', 'table', 'table-cell', 'flex', 'none'];
```

Comment:
For display:none, I think we should still rely on its default display of the element.

For example, `<div style="display:none">` should still be treated as a block, but SPAN should be treated as inline.

So when you see display is none, fallback to its default display:

```ts
const display = element.style.display == 'none' ? null : element.style.display;
const effectiveDisplay = display || getDefaultStyle(element).display || '';

return ...
```